phpunit.xml.dist

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<phpunit colors="true" bootstrap="vendor/autoload.php">
-    <testsuites>
-        <testsuite name="Prophecy PhpUnit Test Suite">
-            <directory suffix="Test.php">./tests/</directory>
-        </testsuite>
-    </testsuites>
-
-    <filter>
-        <whitelist>
-            <directory>./src</directory>
-        </whitelist>
-    </filter>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
+         bootstrap="vendor/autoload.php"
+         cacheResult="false"
+         colors="true"
+>
+  <testsuites>
+    <testsuite name="Prophecy PhpUnit Test Suite">
+      <directory suffix="Test.php">./tests/</directory>
```

Comment:
should be removed or `run_test.php` should be renamed, as it currently matches that suffix on case-insensitive file systems.